utils.py

```diff
@@ -78,21 +90,30 @@ def make_example_shot():
 
 
 def make_uuid():
+    """
+    Create a random UUID.
+    """
     return str(uuid.uuid1()).replace("-", "")
 
 
 def run_in_fresh_loop(coro):
+    """
+    Create a new async event loop.
+    """
     loop = asyncio.new_event_loop()
     task = loop.create_task(coro(loop))
     res = loop.run_until_complete(task)
     loop.close()
     return res
 
 
-def do_login(args):
+def login(args):
+    """
+    Log in or register a new Page Shot "account".
+    """
     global _COOKIES
 
-    async def login(loop):
+    async def _login(loop):
```

Comment:
let's check w/ @tarekziade about the async scenario calling an async function.  should the function be synchronous instead?